src/Raven.Client/Documents/Session/InMemoryDocumentSessionOperations.cs

```diff
@@ -42,6 +42,7 @@ namespace Raven.Client.Documents.Session
     public abstract partial class InMemoryDocumentSessionOperations : IDisposable
     {
         internal long _asyncTasksCounter;
+        internal int? _maxDocsCountOnCachedRenewSession = null;
```

Comment:
Why setting to null by default? In the test you can override it anyway

Reply:
Fixed in commit https://github.com/ravendb/ravendb/pull/11475/commits/61ec6797f4052f5d3b9ca3b9189b1d715d128b44